extensions/src/test/java/org/pentaho/platform/web/http/api/resources/services/SchedulerServiceTest.java

```diff
@@ -1025,6 +1036,288 @@ public void testUpdateBlockoutException() throws Exception {
     verify( schedulerService, times( 2 ) ).addBlockout( jobScheduleRequest );
   }
 
+  @Test
+  public void testHasLogicalRole() throws Exception {
```

Comment:
Like the comments inside this unit test (and others below) indicate, shouldn't this be split into different unit tests?
`testHasLogicalRole_nullUser`, `testHasLogicalRole_nullRole`,...

Reply:
I agree that it should, I was just following the current file structure. By splitting it makes it easier to identify which test failed. I will rearrange the tests I wrote.